src/schema/artwork/index.ts

```diff
@@ -503,6 +504,13 @@ export const artworkFields = () => {
       deprecationReason: "Prefer dimensions instead.",
     },
     meta: Meta,
+    pageviews: {
+      type: GraphQLInt,
+      description: "Weekly pageview data (static).",
```

Comment:
Might be worth it to add a big fat `DO NOT USE` here as well, because deprecation reasons actually don’t surface well in tooling.